lib/movie.dart

```diff
@@ -1,45 +1,66 @@
+import 'genre.dart';
+
 class Movie {
+  final bool adult;
+  final double popularity;
   final int id;
+  int voteCount = 0;
   final String title;
   final String originalTitle;
-  final List<String> genres;
+  final bool video;
+  final List<Genre> genres;
+  final String originalLanguage;
   final String releaseDate;
-  final String voteAverage;
+  final num voteAverage;
   final String overview;
   final String backdrop;
   final String poster;
 
+  void incrementVoteCount() {
+    voteCount++;
+  }
+
   Movie(
+    this.adult,
+    this.backdrop,
+    this.genres,
     this.id,
-    this.title,
+    this.originalLanguage,
     this.originalTitle,
-    this.genres,
+    this.overview,
+    this.popularity,
     this.poster,
-    this.backdrop,
     this.releaseDate,
+    this.title,
+    this.video,
     this.voteAverage,
-    this.overview,
+    this.voteCount,
   );
 
   factory Movie.mockMovie() {
     return Movie(
-      1,
-      'The Super Mario Bros. Movie',
-      'The Super Mario Bros. Movie',
+      false,
+      'https://www.themoviedb.org/t/p/original/9n2tJBplPbgR2ca05hS5CKXwP2c.jpg',
       [
-        'Animation',
-        'Family',
-        'Adventure',
-        'Fantasy',
-        'Comedy',
+        Genre(id: 1, name: 'Animation'),
+        Genre(id: 2, name: 'Family'),
+        Genre(id: 3, name: 'Adventure'),
+        Genre(id: 4, name: 'Fantasy'),
+        Genre(id: 5, name: 'Comedy'),
```

Comment:
genres is a List<int> when you get the data from the genres mock you need to find a way to transform that list<int> on a list<genres>, but leave the model as simple as possible